jarpcbind/src/main/java/org/dcache/jarpcbind/Main.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2009 - 2012 Deutsches Elektronen-Synchroton,
```

Comment:
i copied it from elsewhere. i'll fix this, but you need to update the copyright across youre entire project then :-)